namespaces/referencing-namespace.md

```diff
@@ -0,0 +1,194 @@
+# Referencing Namespace
+
+This namespace defines elements and attributes that may be used to provide the ability to reference and classify elements and namespaces.
+
+This document includes the elements as defined in the Refract specification.
+
+## Referencing and Identifying Elements
+
+These elements provide the means to identify elements with unique or reused qualifiers.
+
+### Ref (Element)
+
+The `ref` element provides a way to reference elements and namespaces with a URI. The context the `ref` element is found in determines what exactly it references. Additionally, the referenced
+
+#### Properties
+
+- `element`: ref (string, required)
+- `content` (string, required) - The URI of the referenced item
+
+### ID (Element)
+
+The `id` element provides a document-wide unique identifier. It MUST be unique for the entire document in which is found. When used as an attribute for another element, it defines the unique ID for that element, and MAY be used in the fragment identifier to link to that element.
```

Comment:
> It MUST be unique for the entire document

I am still having problems with document-unique vs namespace-unique, I would say you id should be namespace-uniqiue and that, in theory, you can have multiple namespaces in a document. Isn't this complicating things tho?